packages/create-react-router/package.json

```diff
@@ -19,7 +19,8 @@
     "./package.json": "./package.json"
   },
   "scripts": {
-    "build": "wireit"
+    "build": "wireit",
+    "typecheck": "tsc"
```

Comment:
We were missing this in a bunch of packages so our root `typecheck` command wasn't getting everything